manifests/init.pp

```diff
@@ -17,18 +17,13 @@
 ) inherits augeas::params {
 
   if versioncmp($::puppetversion, '4.0.0') >= 0 {
-    anchor { 'augeas::begin': }
-    -> class {'::augeas::files': }
-    -> anchor { 'augeas::end': }
+    contain 'augeas::files'
   } else {
-    anchor { 'augeas::begin': }
-    -> class {'::augeas::packages': }
-    -> class {'::augeas::files': }
-    -> anchor { 'augeas::end': }
+    contain 'augeas::packages'
```

Comment:
Perhaps all of this code (puppet < 4) can be removed in a separate PR.  I didn't want to stray too far off my original goal of making sure this module would be compatible with the imminent stdlib 6.0.0

Reply:
LGTM